ans2ask-ui/src/components/Login/Login.jsx

```diff
@@ -26,6 +26,9 @@ export default function Login() {
                     <input type="password" className="form-control bg-lighter" id="password" placeholder="" />
                 </div>
                 <div className="text-center">
+                    <div className="mb-2">
```

Comment:
Great work Miguel! Ideally for web, we should keep the 'forgot password?' and 'Dont have an account' links below the login button. Let's make this change and move lines 28-31 to below the login button initialization.

Reply:
Okay, ty:) now it looks way better!